maven-cache-pmd-plugin/.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,8 @@
+<gradleEnterprise
+        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+    <server>
+        <id>e.grdev.net</id>
+        <url>https://e.grdev.net</url>
+    </server>
+</gradleEnterprise>
```

Comment:
:x: I don't think we should commit this file.

Reply:
Good catch